recipes-wigwag/maestro/maestro_0.0.2.inc

```diff
@@ -0,0 +1,75 @@
+DESCRIPTION = "maestro is a runtime / container manager for deviceOS"
+LICENSE = "Apache-2.0"
+LIC_FILES_CHKSUM = "file://src/${GO_IMPORT}/LICENSE;md5=1dece7821bf3fd70fe1309eaa37d52a2"
+
+GO_IMPORT = "github.com/PelionIoT/maestro"
+
+DEPENDS = "deviceos-users libuv"
+RDEPENDS_${PN} += "bash"
+
+# avoid the `-linkshared` option in this recipe as it causes a panic
+GO_LINKSHARED=""
+
+inherit pkgconfig systemd go gitpkgv edge
+
+RT_SERVICE_FILE = "maestro.service"
+PR = "r0"
+
+SRC_URI = "git://${GO_IMPORT};protocol=https \
+file://${RT_SERVICE_FILE} \
+file://maestro-watcher.service \
+file://maestro-watcher.path \
+"
+
+SRCREV = "v3.0.0"
+
+SYSTEMD_PACKAGES = "${PN}"
+SYSTEMD_SERVICE_${PN} = "maestro.service \
+maestro-watcher.path \
+maestro-watcher.service"
+
+SYSTEMD_AUTO_ENABLE_${PN} = "enable"
+
+RUN_CONFIG_DIR="wigwag/etc/run"
```

Comment:
Can we replace the /wigwag style folder names to an environment variable?

Reply:
@petedyerarm Added new variables to edge bbclass.